lib/Layout/Resolver/TargetType/SingleTaxon.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\Sylius\Layout\Resolver\TargetType;
+
+use Netgen\Layouts\Layout\Resolver\TargetType;
+use Netgen\Layouts\Sylius\Validator\Constraint as SyliusConstraints;
+use Sylius\Component\Taxonomy\Model\TaxonInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Validator\Constraints;
+
+/**
+ * @deprecated this class will be renamed to simply Taxon in next major release
+ */
+final class SingleTaxon extends TargetType
+{
+    public static function getType(): string
+    {
+        return 'sylius_single_taxon';
+    }
+
+    public function getConstraints(): array
+    {
+        return [
+            new Constraints\NotBlank(),
+            new Constraints\Type(['type' => 'numeric']),
+            new Constraints\GreaterThan(['value' => 0]),
+            new SyliusConstraints\Taxon(),
+        ];
+    }
+
+    /**
+     * @return int[]|null
```

Comment:
This PHPDoc is superfluous.

Reply:
@emodric Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/17/commits/057d41669cd3128b87eaf8063b07be5151bc5382